### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by martinglass**
Changes to Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,40 +27,57 @@ A multi-node radio frequency race timing system for FPV drone racing with event
...
 * Multi-frequency node support on Arduino Nanos (track more than one frequency per node).
 * MQTT protocol.
-* Frequency band scanner (UI and standalone tool).
-* Support for server (e.g. Raspberry Pi) audio output (e.g. bluetooth speaker).
-* Link pilots with their MultiGP or iFPV accounts.
-* Sync race events with iFPV.
-* BDRA race formats.
+* BDRA race formats, including "Grace period" (eg 30 seconds) after Time.
+* Command line options to specify configuration file and database.
+* GZip as well as Zip encoding
```
Comment (pulquero) [inline README.md:40]: "GZip encoding of web requests".

File: README.md [hunk 2]
```diff
@@ -27,40 +27,57 @@ A multi-node radio frequency race timing system for FPV drone racing with event
...
+* WiFi OTA support
+* Race metrics
```
Comment (pulquero) [inline README.md:42]: "Pluggable race statistics".

File: README.md [hunk 3]
```diff
@@ -27,40 +27,57 @@ A multi-node radio frequency race timing system for FPV drone racing with event
...
+* Pluggable Leaderboards
+* Test coverage.
+* Continuous Integration (CI)
+* ...and a massive code clean-up!
+
+## Extensions (over RotorHazard 3.0)
+
+* Support for Arduino-based nodes
+* Support for STM32-based nodes
```
Comment (pulquero) [inline README.md:51]: This already existed in 3.0

File: README.md [hunk 4]
```diff
@@ -27,40 +27,57 @@ A multi-node radio frequency race timing system for FPV drone racing with event
...
+* WiFi OTA support
+* Race metrics
+* Pluggable Leaderboards
+* Test coverage.
+* Continuous Integration (CI)
+* ...and a massive code clean-up!
+
+## Extensions (over RotorHazard 3.0)
+
+* Support for Arduino-based nodes
```
Comment (pulquero) [inline README.md:50]: This already existed in 3. Maybe could go in improvements as "Improved I2C handling for Arduino-based nodes".

**Comment by martinglass** [comment]
More work to do yet, so closing this PR now

**Pull request closed without merging**